bcipy/acquisition/tests/test_client_manager.py

```diff
@@ -26,6 +27,7 @@ def setUp(self):
         self.gaze_device_mock.content_type = 'Eyetracker'
         self.gaze_device_mock.sample_rate = 60
         self.gaze_device_mock.is_active = False
+        self.gaze_device_mock.static_offset = 0.0
```

Comment:
It is currently assumed 0.0 for my analyses as well, but I will think of a way to estimate the correct value